src/Components/ReviewList.jsx

```diff
@@ -1,32 +1,46 @@
 import { useEffect, useState } from "react";
 import { ReviewCard } from "./ReviewCard";
 import { getReviews } from "../api";
+import { useSearchParams } from "react-router-dom";
+import { CategoriesNav } from "./Categories";
 
 export const ReviewList = () => {
   const [reviews, setReviews] = useState([]);
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const category = searchParams.get("category");
 
   useEffect(() => {
-    getReviews().then((reviews) => {
+    getReviews(category).then((reviews) => {
       setReviews(reviews);
     });
-  }, []);
+  }, [category, setReviews]);
+
+  const filteredReviews = category
```

Comment:
you API should be doing the filtering for you, so I'm assuming the only reason this is here is because your backend doesn't have a category filter implementation